### PR TITLE
Added test Singleton MSIX package

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Yes, that project name's intentional. The other test MSIX projects are slated to be renamed (sooner or later). This will buy us clarity and consistency w/o colliding with the real package names.

Single it's rarely needed for the forseeable future TB::SetupPackages() only installs it if specified e.g. TB::SetupPackages(TB::Packages::Default | TB::Packages::Singleton). See inc\test\WindowsAppSDK.Test.Bootstrap.h

### CODE REVIEW

**Review by EHO-makai — APPROVED** [review]

File: test/DynamicDependency/data/WindowsAppSDK.Test.Singleton.Msix/WindowsAppSDK.Test.Singleton.Msix.vcxproj [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
```
[the remark this answers is not in the document]
Comment (DrusTheAxe) [reply test/DynamicDependency/data/WindowsAppSDK.Test.Singleton.Msix/WindowsAppSDK.Test.Singleton.Msix.vcxproj:4]: Not in Helix yet; @alexlamtest is working on it.

But this _is_ bugged as we do _build_ for ARM64. I missed adding that when I created the project (VS' templates are poor fits for us making WinAppSDK; we really need investment here @BenJKuhn hint hint...)

**Pull request merged**